Martinh/relayer sdk note (#358)
wormhole-foundation/wormhole-docs: build/infrastructure/relayers/run-relayer.md

## Use the Wormhole SDK

!!! note   

>     The example below uses the older [`@certusone/wormhole-sdk`](https://www.npmjs.com/package/@certusone/wormhole-sdk){target=\_blank}, which is still supported and used in the Relayer Engine but is no longer actively maintained.
>     
>     For most use cases, we recommend the latest [`@wormhole-foundation/sdk`](https://www.npmjs.com/package/@wormhole-foundation/sdk){target=\_blank}.

Reviewer: ```suggestion
    The example below uses the legacy [`@certusone/wormhole-sdk`](https://www.npmjs.com/package/@certusone/wormhole-sdk){target=\_blank}, which is still supported and used in the Relayer Engine but is no longer actively maintained.
    
    For most use cases, it is recommend to use the latest [`@wormhole-foundation/sdk`](https://www.npmjs.com/package/@wormhole-foundation/sdk){target=\_blank}.
```

avoid we
